Одиннадцатая лабораторная работа выполнена

PopovMatvey opened the pull request:
Был свёрстан слайдер, согласно требованиям.

Прикладываю скриншот выполненного задания:
![Slider-for-PR](https://user-images.githubusercontent.com/85041400/164885829-813552e8-762c-45ef-a4d7-ec74255fdb01.png)

PopovMatvey committed:
Create task-01.css

PopovMatvey committed:
Create first.jpg

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Create task-01.js

PopovMatvey committed:
Одиннадцатая лабораторная работа выполнена

Выполнены все требования, согласно заданию.

djilua1 commented on src/ex11_js_dom/assets/js/task-01.js:
```diff
@@ -0,0 +1,61 @@
+sliderLine.appendChild(prevButton);
+
+nextButton.className = 'next';
+nextButton.innerHTML = '&#10095;';
```
Это можно в html сделать.

djilua1 commented on src/ex11_js_dom/assets/js/task-01.js:
```diff
@@ -0,0 +1,61 @@
+    sliderLine.appendChild(imgBlock);
+    imgBlock.src = inputArray[i];
+  }
+}
```
Можно по одной отрисовывать.

djilua1 commented on src/ex11_js_dom/assets/js/task-01.js:
```diff
@@ -0,0 +1,61 @@
+const sliderLine = document.querySelector('.slider');
+const prevButton = document.createElement('button');
+const nextButton = document.createElement('button');
+const arrayImages = ['assets/images/first.jpg', 'assets//images/second.jpg', 'assets//images/third.jpg'];
```
Импорты отбиваем пустой строкой.

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Add files via upload

PopovMatvey committed:
Исправления по одиннадцатой лабораторной работе

Были исправлены три замечания:
1. Кнопки теперь не создаются в js, они заданы сразу в html;
2. Изображения теперь отображаются по одному и display: none; для их скрытия не используется;
3. Импорты элементов html отбиты пустой строкой от переменных js.

djilua1 commented on src/ex11_js_dom/assets/js/task-01.js:
```diff
@@ -0,0 +1,60 @@
+const sliderLine = document.querySelector('.slider');
```
У тебя же даже подсвкчивается тут ошибка. О том, что эта переменная нигде не используется.

djilua1 commented on src/ex11_js_dom/assets/js/task-01.js:
```diff
@@ -0,0 +1,60 @@
+let defaultSlideIndex = 1;
+
+function createAllImages(inputArray) {
+  for (let i = 0; i < inputArray.length; i++) {
```
В таких случаях либо отключай правило, либо дизейбли линтер.

djilua1 commented on src/ex11_js_dom/assets/js/task-01.js:
```diff
@@ -0,0 +1,60 @@
+
+  slides[defaultSlideIndex - 1].animate([
+    { opacity: '0.4' },
+    { opacity: '1' }
```
Висячая запятая.
